Trakt/ServerMediator.cs

```diff
@@ -314,146 +314,65 @@ private async void KernelPlaybackProgress(object sender, PlaybackProgressEventAr
-                        }
-                        else if (video is Episode episode)
-                        {
-                            _playbackState[traktUser.LinkedMbUserId].PlaybackProgress = playbackProgressEventArgs.PlaybackPositionTicks ?? 0L;
-                            _playbackState[traktUser.LinkedMbUserId].IsPaused = false;
-                            _playbackState[traktUser.LinkedMbUserId].PlaybackTime = DateTime.UtcNow;
-
-                            _logger.LogDebug("Sending episode playback status update to trakt.tv for user {User}.", user.Username);
-                            await _traktApi.SendEpisodeStatusUpdateAsync(
-                                episode,
-                                MediaStatus.Watching,
-                                traktUser,
-                                progressPercent).ConfigureAwait(false);
-                        }
-                    }
-                    else if (tickDifferenceInSeconds < 0 || tickDifferenceInSeconds > realTimeDifferenceInSeconds + 15)
-                    {
-                        _logger.LogDebug("Playback skipped");
-                        if (video is Movie movie)
-                        {
-                            _playbackState[traktUser.LinkedMbUserId].PlaybackProgress = playbackProgressEventArgs.PlaybackPositionTicks ?? 0L;
-                            _playbackState[traktUser.LinkedMbUserId].IsPaused = false;
-                            _playbackState[traktUser.LinkedMbUserId].PlaybackTime = DateTime.UtcNow;
-
-                            _logger.LogDebug("Sending movie playback status update to trakt.tv for user {User}.", user.Username);
-                            await _traktApi.SendMovieStatusUpdateAsync(
-                                movie,
-                                MediaStatus.Watching,
-                                traktUser,
-                                progressPercent).ConfigureAwait(false);
-                        }
-                        else if (video is Episode episode)
-                        {
-                            _playbackState[traktUser.LinkedMbUserId].PlaybackProgress = playbackProgressEventArgs.PlaybackPositionTicks ?? 0L;
-                            _playbackState[traktUser.LinkedMbUserId].IsPaused = false;
-                            _playbackState[traktUser.LinkedMbUserId].PlaybackTime = DateTime.UtcNow;
-
-                            _logger.LogDebug("Sending episode playback status update to trakt.tv for user {User}.", user.Username);
-                            await _traktApi.SendEpisodeStatusUpdateAsync(
-                                episode,
-                                MediaStatus.Watching,
-                                traktUser,
-                                progressPercent).ConfigureAwait(false);
-                        }
-                    }
+                    _logger.LogDebug("Playback state did not change.");
```

Comment:
Not everyone has scrobbling enabled. Shouldn't it update the progress on Trakt when it's not paused?

Reply:
I don't really understand what you mean. If you disable scrobbling playback is not synced at all. If you start or resume playback we send one request indicating watching state to trakt. After that trakt automatically progresses the playtime until either the end is reached or we indicated paused or finished playback.